vars/RedHat.yml

```diff
@@ -28,3 +28,6 @@ __php_opcache_conf_filename: 10-opcache.ini
 __php_fpm_daemon: php-fpm
 __php_fpm_conf_path: "/etc/fpm"
 __php_fpm_pool_conf_path: "/etc/php-fpm.d/www.conf"
+
+__php_fpm_pool_user: apache
```

Comment:
Can we also have the variable for the group.  www.conf has both user and group values, so it would be nice to be able to control both and not just the user half. :smile: